File: 3379-ScoreOfAString/3379-ScoreOfAString.py
# Last updated: 8/13/2025, 8:15:26 PM
class Solution:
    def scoreOfString(self, s: str) -> int:
        res = 0
        for i in range(1,len(s)):
            res += abs(ord(s[i-1]) - ord(s[i]))
        
        return res

File: 3379-ScoreOfAString/3379-ScoreOfAString.js
// Last updated: 8/13/2025, 8:15:34 PM
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let sum = 0
    for(let i = 1; i< s.length ; i++){
        sum = sum + Math.abs(s[i-1].charCodeAt() - s[i].charCodeAt())
    } 
    return sum
};